src/app/helptext/system/failover.ts

```diff
@@ -23,17 +24,26 @@ export const helptext_system_failover = {
     sync_to_message: T("Sync to peer succeeded."),
   },
 
-  disabled_placeholder: T('Disabled'),
-  disabled_tooltip: T(''),
+  disabled_placeholder: T('Disable Failover'),
+  disabled_tooltip: T('Disables automatic failover and changes the \
+ <i>HA Enabled</i> icon to <i>HA Disabled</i>.'),
 
-  master_placeholder: T('Master'),
-  master_tooltip: T(''),
+  master_placeholder: T(`Default ${globalHelptext.ctrlr}`),
+  master_tooltip: T(`Marks the currently active ${globalHelptext.ctrlr} \
```

Comment:
Present tense.  "Marks" is a little odd.
```suggestion
  master_tooltip: T(`Make the currently active ${globalHelptext.ctrlr} \
```